### PR TITLE
Fix cmsUnload

### PR DESCRIPTION
**Pull request by pcanal**

### CODE REVIEW

**Comment by Axel-Naumann** [comment]
The error was introduced by a commit here https://epsft-jenkins.cern.ch/job/root-nightly-master/2198/LABEL=ROOT-ubuntu18.10,SPEC=cxx17/testReport/projectroot.roottest.root.meta/cmsUnload/roottest_root_meta_cmsUnload_make/ (for v6-18). How do the failure, these commits and your fix correlate?

**Comment by Axel-Naumann** [comment]
Rumors say that making MLP optional (and not enabling it in Jenkins) triggered this. This PR indeed fixes the issue; after the successful PR builds here I have enabled MLP in Jenkins.

File: interpreter/cling/include/cling/Interpreter/Transaction.h [hunk 1]
```diff
@@ -450,6 +450,7 @@ namespace cling {
     ///
     void clear() {
       m_DeclQueue.clear();
+      m_DeserializedDeclQueue.clear();
```
Comment (Axel-Naumann) [inline interpreter/cling/include/cling/Interpreter/Transaction.h]: Yikes! Thanks!

**Comment by amadio** [comment]
If you look at my PR (#3917), you can see that the test failed in the initial build, but when I asked to build "with flags -Dmlp=ON" it stopped failing. The Jupyter failure was due to a missing package on the new node.

File: interpreter/cling/include/cling/Interpreter/Transaction.h (hunk 1 above)
Comment (pcanal) [reply interpreter/cling/include/cling/Interpreter/Transaction.h]: But it was not causing any problem since this routine is not used ...

**Pull request merged**